### PR TITLE
Feature/fix bug all code

### PR DESCRIPTION
**Pull request by DuongPham-dev**
- Refactor and update code

### CODE REVIEW

File: practices/practice-one/src/scripts/helpers/validate.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    async validateCode(code) {
+        const service = new Service();
+        service.setParams({
+            code,
+        });
+        const respone = await service.request();
+        const length = respone.data.length;
+        const codeLength = code.length;
+        const valid = !Boolean(code)
+            ? { code: "Can't be left blank" }
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/scripts/helpers/validate.js]: Add this into as constant

Comment (DuongPham-dev) [reply practices/practice-one/src/scripts/helpers/validate.js]: I still can't follow you.

Comment (cunguyen-agilityio) [reply practices/practice-one/src/scripts/helpers/validate.js]: Handle like this, the import to use 

```
export default {
   INVALID_CODE: "Can't be left blank"
}
```

Comment (DuongPham-dev) [reply practices/practice-one/src/scripts/helpers/validate.js]: i understood

File: practices/practice-one/src/index.html [hunk 2]
```diff
@@ -5,7 +5,12 @@
...
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <link rel="stylesheet" href="./styles/main.css" />
-        <title>Document</title>
+        <link
+            rel="icon"
+            href="https://websitehoctructuyen.com/wp-content/uploads/2020/10/phan-mem-quan-ly-hoc-sinh.jpg"
+            type="image/x-icon"
+        />
+        <title>Student manager</title>
         <style></style>
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html:10]: Why?

File: practices/practice-one/src/index.html [hunk 3]
```diff
@@ -5,7 +5,12 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <link rel="stylesheet" href="./styles/main.css" />
-        <title>Document</title>
+        <link
+            rel="icon"
+            href="https://websitehoctructuyen.com/wp-content/uploads/2020/10/phan-mem-quan-ly-hoc-sinh.jpg"
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html]: Please download then use as local file

Comment (DuongPham-dev) [reply practices/practice-one/src/index.html]: Ok

File: practices/practice-one/src/index.html (hunk 2 above)
Comment (DuongPham-dev) [reply practices/practice-one/src/index.html:10]: I don't know what you' re talking about

**Pull request merged**